### ISSUE TITLE
WCAG 2.5.5: Touch inputs must have a sufficient target size. (android.widget.Button)

### ISSUE THREAD

**Issue by SuryacharanM** [dotnet/maui#6597]
GitHubTags:
#A11yAuto; #A11yUsable; #.NET MAUI; #AILimited; #A11ySev4; #A11yTCS; #Android; #AndroidPhone; #GH_.NETMAUI_Android_Apr2022; #Accessibility; #Accessibility Insights for Android; #WCAG 2.5.5;, #TouchSizeWcag;

### Issue: 
Touch inputs must have a sufficient target size. (TouchSizeWcag - https://accessibilityinsights.io/info-examples/android/touch-size-wcag/)

### Target application: 
com.microsoft.maui.sample

### Element path: 
android.widget.Button

### How to fix: 
The element has an insufficient target size (width: 26.571dp, height: 184dp). Set the element's minWidth and minHeight attributes to at least 44dp.

### Environment: 

====

This accessibility issue was found using Accessibility Insights for Android 2021.714.15 (axe-android), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.

### Steps to Reproduce

1. Install and open ".NET MAUI" app.
2. "Home Screen" will open.
3. Check for the Accessibility Insights and observe the issue.
![image](https://user-images.githubusercontent.com/86710788/165719996-9e6e33aa-0fbb-4175-9b77-90b273dc92ef.png)

### Version with bug

Unknown/Other (please specify)

### Last version that worked well

Unknown/Other

### Affected platforms

Android

### Affected platform versions

Android 11

### Relevant log output

```shell
The element has an insufficient target size (width: 26.571dp, height: 184dp). Set the element's minWidth and minHeight attributes to at least 44dp.
```

**Comment by rachelkang** [comment]
Hi, @SuryacharanM - thanks for filing this issue! As can be seen, each row of buttons is laid out in a horizontal ScrollView. This mean that scrolling horizontally will render the full size of the button which is of sufficient target size. Whenever a ScrollView is used and the content takes up more space, part of the content will appear off-screen until it is scroled to.

Would making the scroll bar more prominent suffice? Is it ok to leave this as it is? What would you recommend as the best approach to resolving this issue?

**Comment by keerthiduvvuri** [comment]
Hi @rachelkang i have verified this issue in ".NET MAUI" App, Version 1.0.0
The issue is reproducing still. Can you please provide us version details in which these are fixed. 
Thanks.

**Comment by keerthiduvvuri** [comment]
Hi @rachelkang This is Sev4 bug, as per above comment you can leave as it is, you can close the bug.
Thanks.
